app/forms/state_file/nc_subtractions_form.rb

```diff
@@ -0,0 +1,27 @@
+module StateFile
+  class NcSubtractionsForm < QuestionsForm
+    set_attributes_for :intake, :tribal_member, :tribal_wages_amount
+
+    validates :tribal_member, inclusion: { in: %w[yes no], message: :blank }
+    validates_numericality_of :tribal_wages_amount, if: -> { tribal_member == "yes" }
+    validates :tribal_wages_amount, presence: true, allow_blank: false, numericality: { greater_than_or_equal_to: 1 }, if: -> { tribal_member == "yes" }
+    validate :below_1040_amount, if: -> { tribal_wages_amount.present? }
+
+    def save
+      if tribal_member == "no"
+        @intake.update(tribal_member: "no", tribal_wages_amount: nil)
+      else
+        @intake.update(attributes_for(:intake))
+      end
+    end
+
+    private
+
+    def below_1040_amount
```

Comment:
why is it called `1040_amount`? does 1040 mean `fed_wages_salaries_tips`?

Reply:
yes its from the federal form

Reply:
is 1040 the form name which includes `fed_wages_salaries_tips`? I'd rather call this something more like `below_fed_wages_salaries_tips` but is there a reason to point to 1040?